config/routes.rb

```diff
@@ -368,6 +368,7 @@
       get '/session/errors/ssn_failure' => 'session_errors#ssn_failure'
       get '/session/errors/exception' => 'session_errors#exception'
       get '/session/errors/throttled' => 'session_errors#throttled'
+      get '/session/errors/rate_limited' => 'session_errors#throttled'
```

Comment:
You can probably go ahead and rename the action in SessionErrorsController as well?

```suggestion
      get '/session/errors/throttled' => 'session_errors#rate_limited'
      get '/session/errors/rate_limited' => 'session_errors#rate_limited'
```

Reply:
Good idea. https://github.com/18F/identity-idp/pull/8886/commits/bca4111daf8e53ba511c69b026583ef299c65ba0 There are also some javascript references to 'throttled' I'm not sure how to change. Maybe we can take a look on Monday.